### DISCUSSION TITLE
Players that support <podcast:liveItem>

### DISCUSSION THREAD

**Discussion by redimongo** [Podcastindex-org/podcast-namespace#424] [Q&A] [answered]
Out of interest I am wondering what players currently support <podcast:liveItem>
I can see a number of issues with this tag and I am trying to understand how if you have a podcast that has the <item> tag in it would you add this as another tag and it would still be valid?

I also ask this from a media perspective because it could come in handy for radio shows, if we can trigger this tag and then our shows can go live on many apps not just our app.

**Comment by jamescridland** [comment, marked as answer]
There's one, currently: Podverse. But it's very, very new. [This is the full list](https://podcastindex.org/apps?appTypes=app&elements=Live), filtered for support of the live tag.

Github has mangled your question, so not quite sure what you mean: but for a radio station's use, there's a working example for the [100% Retro](https://100percentretro.com/) station - [you can find that RSS feed here](https://feeds.podcastindex.org/100retro.xml).

Here's the [code to produce a liveItem podcast feed](https://github.com/Podcastindex-org/100retro) from @daveajones

**Comment by SpencerPearson** [comment]
@redimongo CurioCaster also supports the liveItem:

https://curiocaster.com

I'm hearing whispers that Fountain is also working on adding liveItem implementation.

We use the liveItem to describe our live episodes, which is definitely a different event and experience than that of the published show, and with a different audience as well. For us it makes perfect sense to have the liveItem tags because of those factors. There are several ways to do it, but we keep two liveItem tags in our feed at all times because of our format. There is one for our weekly Tuesday night show and one describing the next or last (or current, while it's on the air) Bowls With Buds interview format that we schedule more irregularly. Here's my process for each:

For the Tuesday show, we flip the status from "pending" to "live" right as we get ready to start the show, and we send out a Podping announcement that we're live. Once the show ends, I briefly change the status to "ended" while I work on post-production, and then shortly thereafter (a couple hours or less) I update that liveItem to reflect the upcoming Tuesday show and set its status to "pending" again.

For the Bowls With Buds liveItem, I leave the most recent episode as "ended" until I have another guest scheduled. Once I have the guest's date & time locked in, I gather the info for their person tag (proper name attribution, a thumbnail image and the site they want to be linked) as well as info for their value split (if they don't have a lightning node of their own we encourage them to get an Alby wallet and send us their "customValue" from the bottom of https://getalby.com/node once they have set up an account).

liveItem has been in use since @daveajones visited us for a Bowls With Buds last February, so we're coming up on the first anniversary of liveItem implementation and thus @jamescridland is correct in that it's a relatively new part of the namespace still. I think liveItem has tremendous potential for those of us who produce live-to-tape and cater more toward the live audiences. Here's a nice write-up @daveajones did recently on liveItem implementation:

https://podcasting20.substack.com/p/live-podcasting-the-open-way
